### ISSUE TITLE
which language is it for？

### ISSUE THREAD

**Issue by LiYijin** [zzw922cn/Automatic_Speech_Recognition#9]
Sorry I am a freshman in speech recognition
I saw that "timit phonemes, it is 62; if timit characters, it is 29"
I want to know is it for English?
If for chinese, the number of phonemes and characters should be how many?
really thanks!

**Comment by brianlan** [comment]
@LiYijin It's English.

**Comment by brianlan** [comment]
@zzw922cn regarding this, I also have a question. The num_classes for 'cha' is 29. But why it's 29 not 28? According to my understanding, we only have 26 letters + 2 special char (space and single-quote), right? I can see from [this line of code](https://github.com/zzw922cn/Automatic_Speech_Recognition/blob/master/src/utils/utils.py#L126), when ind == 28, it just passed.

**Comment by zzw922cn** [comment]
@LiYijin @brianlan Yes, my code is currently for English, but I would support Chinese in future. For Chinese, you can also get the phenome of a word, for example, the phoneme sequence of '安宁' is 'aa an1 n ing2'.

**Comment by LiYijin** [comment]
@brianlan  @zzw922cn  Thanks for you two.
I know we can get the phenome of a chinese word, 
But I am still confused that  I think it is a standard for the number of phonemes and characters.
For chinese, the standard number of phonemes should be how many? or where could I find it?

**Comment by mnottheone** [comment]
@brianlan regarding 29th character, for decoding in general to handle repetition one extra character is used. Most of places it's character set + '_' . I will suggest you to read decoding section and pseudo code given [here](http://deeplearning.stanford.edu/lexfree/lexfree.pdf) . Hope, it helps with your doubt.
